docs/ReleaseNotes.md

```diff
@@ -25,7 +25,7 @@ The Apache Commons library has been removed as a dependency. There were a few lo
 * **Performance** Improvement 3 [(Issue #NNN)](https://github.com/FoundationDB/fdb-record-layer/issues/NNN)
 * **Performance** Improvement 4 [(Issue #NNN)](https://github.com/FoundationDB/fdb-record-layer/issues/NNN)
 * **Performance** Improvement 5 [(Issue #NNN)](https://github.com/FoundationDB/fdb-record-layer/issues/NNN)
-* **Feature** Feature 1 [(Issue #NNN)](https://github.com/FoundationDB/fdb-record-layer/issues/NNN)
+* **Feature** Add tests for Lucene "all-match" scan [(Issue #2991)](https://github.com/FoundationDB/fdb-record-layer/issues/2991)
```

Comment:
Technically, only user-visible changes require updating release notes: https://github.com/FoundationDB/fdb-record-layer/blob/main/CONTRIBUTING.md#opening-a-pull-request
The only harm of doing it excessively is more conflicts, so not worth removing, just for future reference.

Reply:
Hmm, that's interesting - I was doing that for all changes (this way the issue gets closed too), but point taken

Reply:
The issue gets closed from the commit message, it does not look at release notes.